assets/scss/app.scss

```diff
@@ -7,7 +7,8 @@
  * file that was distributed with this source code.
  */
 
-// Eonasdan Bootstrap DateTimePicker in its version 3 does not
-// provide the scss or plain css, it only provides the less version
-// of its source files, that's why it is not included it via npm.
-@import '../vendor/bootstrap-datetimepicker.min.css';
+@import '@eonasdan/tempus-dominus';
+
+html {
+  font-size: 100% !important;
```

Comment:
This should probably be moved to sonata, for some reason, our version of admin-lte, seets font-size to 10px globally on html, and that is not good idea because it messes up with the rem size.